tools/utils/build_tools.py

```diff
@@ -130,10 +131,23 @@ def get_usd_header_info(usd_include_dir):
     r = re.search('#if ([01]).#define PXR_PYTHON_SUPPORT_ENABLED', pxr_h, re.DOTALL)
     if r:
         HAS_PYTHON_SUPPORT = r.group(1) == '1'
-
+        
+    # Detect updated Compositor
+    compositor_h_path = os.path.join(usd_include_dir, "pxr", "imaging", "hdx", "compositor.h")
```

Comment:
This needs to work if compositor.h does not exist and default to False. `get_usd_header_info` runs for all builds, even when the render delegate is not built, or when USD does not have its imaging component built. We don't need to print any warnings, so a simple try/except with pass for the exception should be enough.

Reply:
Should be fixed with https://github.com/Autodesk/arnold-usd/pull/64/commits/5c41639839eae9f7fb5aa8260408e8d92a3f8449.

Reply:
Yep, this looks good, thanks!